.github/workflows/scala.yml

```diff
@@ -9,9 +7,20 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - run: git fetch --tags --unshallow
     - name: Set up JDK 1.8
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
     - name: Run tests
       run: sbt +test scripted
+    - uses: olafurpg/setup-gpg@v2
```

Comment:
who owns this? do we trust them with our gpg keys?

Reply:
it's the developer of the sbt-ci-release plugin, so yes...